src/components/RandomUserApp.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import Button from "./Button";
+import { UsersNameList } from "./UsersNameList";
+import Card from './Card';
+import Loader from 'react-loader-spinner';
+
+const RandomUserApp = () => {
+  const [users, setUsers] = useState([]);
+  const [selectedUser, setSelectedUser] = useState({});
```

Comment:
When you first load the page there is no user selected. Also when you click the *Get Users* button again, the previously selected user is still displayed.  
You might want to consider adding a `useEffect` here to update the selected user once the users array is changing and on load select the first user.